pdp/main.py

```diff
@@ -56,6 +57,7 @@ def initialize_frontend(global_config, use_auth=False, use_analytics=False):
     wsgi_app = PathDispatcher([
         ('^/css/(default|pcic).css$', static.Cling(resource_filename('pdp_util', 'data'))), # a bit of a hack for now
         ('^/check_auth_app/?$', check_auth),
```

Comment:
With /user/profile is the `check_auth_app` now obsolete/unnecessary? If so, it should be removed from the code and cleaned up.